src/renderer/components/GenericModal/_css_variables.scss

```diff
@@ -0,0 +1,113 @@
+:root {
+    // Default Hex values to be replaced by applyTheme()
+    // these are define to prevent flashes of unstyled content
+    --icon-opacity: 0.64;
+    --icon-opacity-hover: 0.8;
+    --away-indicator: #ffbc42;
```

Comment:
Does the desktop app need to define its own values for these so that the new modal follows its theme?

Reply:
The theme is actually set by a separate module called `bootstrap-dark` that injects dark mode styles when it's enabled, so leaving these as-is should be fine.

Reply:
Will it override the values of these variables though? Bootstrap may have its own theme variables, but they could clash with these ones from the web app

Reply:
It won't change our variables, no. I forget exactly what it does but I think it's something to do with just lazy loading the style sheets. The way it works for us is that when the `darkMode` class is applied it will use styles that use the variables we have.